fix: data issues after check 01.06.2022

cehali opened the pull request.

hatskier commented on manifests/ascendex.json:
```diff
@@ -21,7 +21,6 @@
     "XRP": {},
```
Can you please add a short summary of the fixes in the PR description?
Also, why have you removed several pangolin-based fetchers? Didn't all of them have correct pricing data? I am asking about this specifically, because they were used in production by DeltaPrime protocol :)

cehali replied:
Here you have responses from different pangolin with missing symbols: https://github.com/redstone-finance/redstone-node/issues/91#issue-1255823222

cehali commented:
Fix: problems with ascendex -> Removed USDC/USDT pair, rest of them started to work, EDIT: cleaned few another coins, one coin failing was disturbing whole ascendex

Fix: check why there are no valid values for symbols: HPB, TVK, UMB, OLT, CET, AGS, JRT, SMTY, SHOPX, HOPR, PHX, UNN, DFD, ALH -> All of them beside ALH fixed when a problem with ascendex disappeared

ALH is no longer supported on coingecko

crex24 seems to be unavailable or corrupted. In Poland I cannot open their page because they say is not available here and responses from there are almost always a lot different than others
image
image
image

RSR - values on sushiwap and uniswap still a lot off from others

UMA - on probit constantly value really high

USDT - low value on sushiswap

BZRX - low value on sushiswap and high on bitcoincom hitbtc

SKM - big value on bittrex

UST - low value on uniswap

ESD - it is only from uniswap and sushiswap and they have different values from each other

For now didn't remove coins with big derivations

Fix: why not included in response - a lot of cleaning there, removed coins which were providing errors.
